docs/changelog.rst

```diff
@@ -2,7 +2,7 @@
 Changelog
 =========
 
-* :release:`1.35.0 <2024-10-01>`
+* :release:`1.35.0 <2024-10-02>`
```

Comment:
![imagen](https://github.com/user-attachments/assets/c5b0515f-d48d-48f9-8537-24ba0328f2f4)

Reply:
hehe yeah not happening today. Multiple open things